spec/factories/preferences.rb

```diff
@@ -16,8 +16,9 @@
 #
 FactoryBot.define do
   factory :preference do
-    name { "MyString" }
-    description { "MyText" }
+    name { 'MyString' }
```

Comment:
What about adding some randomness with [faker](https://github.com/faker-ruby/faker) gem?
```suggestion
    name { Faker::Name.unique.name }
```